CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG for mcrouter
 
+## [v3.0.0] (2017-08-15)
+
+### Breaking changes:
+* Removed support for Ubuntu 14.04
+* Converted `node['folly']['version']` to `node['mcrouter']['commit_hash']` given that FB has a weird release process.
```

Comment:
Oh ok, so the release process I was referring to was `mcrouter`. You can see that `mcrouter` includes a FOLLY_COMMIT and WANGLE_COMMIT file in its repo:

https://github.com/facebook/mcrouter/commit/e907287079697baed001eedcac1194d7eeb86991

So the build process, according to their own scripts, is to read that commit hash and checkout out that specific version, you can see that here for Folly
https://github.com/facebook/mcrouter/blob/master/mcrouter/scripts/recipes/folly.sh#L8
and here for Wangle:
https://github.com/facebook/mcrouter/blob/master/mcrouter/scripts/recipes/wangle.sh#L8

Removing that `[folly][version]` attribute had the idea of "as a user of this cookbook you don't have to worry about the commit hashes, because Facebook already provides that in their own repo" if that makes sense.